examples/text_labels_and_annotations/accented_text.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+r"""
+=================================
+Using accented text in matplotlib
+=================================
+
+Matplotlib supports accented characters via TeX mathtext or unicode.
+
+Using mathtext, the following accents are provided: \hat, \breve, \grave, \bar,
+\acute, \tilde, \vec, \dot, \ddot.  All of them have the same syntax,
+e.g., to make an overbar you do \bar{o} or to make an o umlaut you do
+\ddot{o}.  The shortcuts are also provided, e.g.,: \"o \'e \`e \~n \.x
+\^y
+"""
+from __future__ import unicode_literals
+import matplotlib.pyplot as plt
+
+# Mathtext demo
+fig, ax = plt.subplots()
+ax.plot(range(10))
+ax.set_title(r'$\ddot{o}\acute{e}\grave{e}\hat{O}'
+             r'\breve{i}\bar{A}\tilde{n}\vec{q}$', fontsize=20)
+
+# Shorthand is also supported and curly's are optional
+ax.set_xlabel(r"""$\"o\ddot o \'e\`e\~n\.x\^y$""", fontsize=20)
+fig.tight_layout()
+
+# Unicode demo
+fig, ax = plt.subplots()
+ax.set_title('Développés et fabriqués')
+ax.set_xlabel("réactivité nous permettent d'être sélectionnés et adoptés")
```

Comment:
Actually both French sentences are meaningless.  If we want to keep some foreign languages, I can propose
```
GISCARD CHAHUTÉ À L'ASSEMBLÉE
LE COUP DE DÉ DE DE GAULLE
```
(famous examples where keeping accents on caps is needed for the sentence to make any sense) :-)